### PR TITLE
 added sandbox key to toggle dev/prod envs

### PR DESCRIPTION
**Pull request by manuelgeek**
Solution
Removed the `force-liveurl` key and added `sandbox` key to the `ex_pesa` configs. The key is a boolean to determine whether the keys added are for sandbox or live.

Re read on the README configs to understand better

### CODE REVIEW

**Review by lenileiro — APPROVED** [review]
LSTM

File: lib/ex_pesa/util.ex [hunk 1]
```diff
@@ -5,8 +5,8 @@ defmodule ExPesa.Util do
   @spec get_url(String.t(), String.t()) :: String.t()
   def get_url(live_url, sandbox_url) do
     cond do
-      Mix.env() == :prod -> live_url
-      Application.get_env(:ex_pesa, :force_live_url) == "YES" -> live_url
+      Application.get_env(:at_ex, :sandbox) === false -> live_url
```
Comment (lenileiro) [inline lib/ex_pesa/util.ex:8]: Change at_ex to ex_pesa

**Review by lenileiro — CHANGES_REQUESTED** [review]

File: lib/ex_pesa/util.ex (hunk 1 above)
Comment (manuelgeek) [reply lib/ex_pesa/util.ex]: done, nice catch

**Review by lenileiro — APPROVED** [review]
@manuelgeek nice work

**Pull request merged**